joinrun/src/test/scala/code/chymyst/test/StaticAnalysisSpec.scala

```diff
@@ -169,7 +169,7 @@ class StaticAnalysisSpec extends FlatSpec with Matchers with TimeLimitedTests {
     val possibleErrors = Set(("45EA", "465D", "4BE5", "465D"), ("4AEA", "A89F", "B07E", "4AEA")).map(tup4 =>
       s"In Site{a + a + a + b + b + b + b => ...; a + a + a + b + b + b => ...}: Unavoidable nondeterminism: reaction"+
         s" {a(<${tup4._1}...>) + a(<${tup4._2}...>) + a(<${tup4._3}...>) + b(1) + b(1) + b(2) + b(_) => }" +
-        s" is shadowed by {a(<${tup4._4}...>) + a(_) + a(.) + b(1) + b(1) + b(_) => }")
+        s" is shadowed by {a(<${tup4._4}...>) + a(_) + a(x) + b(1) + b(1) + b(_) => }")
```

Comment:
I'm not sure this will work... and it's very tedious to switch between scala 2.11 and 2.12.